website/docs/faqs/Accounts/delete-users

```diff
@@ -0,0 +1,19 @@
+---
+title: How do I delete a user in dbt Cloud?
+description: "Deleting a user in dbt Cloud"
+sidebar_label: 'How to delete a user'
+id: delete-a-user
```

Comment:
i believe the id should match the file name - is that right? this faq isn't showing up under **Accoutns** when you go to the deploy preview. https://deploy-preview-2535--docs-getdbt-com.netlify.app/faqs/Accounts/transfer-account

```suggestion
id: delete-users
```